configure-clusters/src/main/java/com/esri/arcgismaps/sample/configureclusters/screens/MainScreen.kt

```diff
@@ -0,0 +1,413 @@
+            modifier = Modifier.padding(8.dp)
+        )
+        var expanded by rememberSaveable { mutableStateOf(false) }
+        Box(
+            modifier = Modifier
+                .wrapContentHeight()
+                .wrapContentWidth()
+                .wrapContentSize(Alignment.TopStart)
+                .padding(8.dp)
+        ) {
+            ExposedDropdownMenuBox(
+                expanded = expanded,
+                onExpandedChange = { expanded = !expanded }
+            ) {
+                TextField(
+                    value = clusterMaxScale.toString(),
+                    onValueChange = {},
+                    readOnly = true,
+                    trailingIcon = { ExposedDropdownMenuDefaults.TrailingIcon(expanded = expanded) },
+                    modifier = Modifier.menuAnchor()
+                )
+                ExposedDropdownMenu(
+                    expanded = expanded,
+                    onDismissRequest = { expanded = false }
+                ) {
+                    clusterMaxScaleOptions.forEachIndexed { index, clusterRadius ->
+                        DropdownMenuItem(
+                            text = { Text(clusterRadius.toString()) },
+                            onClick = {
+                                updateClusterMaxScale(
+                                    index
+                                )
+                                expanded = false
+                            })
+                        // show a divider between dropdown menu options
+                        if (index < clusterMaxScaleOptions.lastIndex) {
+                            Divider()
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+/**
+ * Composable function to display the cluster info content from the pop up within a bottom sheet.
+
+ */
+@Composable
+private fun ClusterInfoContent(
+    popUpTitle: String,
+    popUpInfo: Map<String, Any?>,
+    onDismiss: () -> Unit
+) {
+    Column(Modifier.background(MaterialTheme.colorScheme.background)) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Text(
+                modifier = Modifier
+                    .padding(horizontal = 30.dp, vertical = 12.dp)
+                    .weight(6f),
+                text = popUpTitle,
+                style = SampleTypography.displaySmall
+            )
+
+            IconButton(
+                modifier = Modifier.weight(1f),
+                onClick = onDismiss
+            ) {
+                Icon(
+                    imageVector = Icons.Rounded.Close,
+                    contentDescription = "Close button"
+                )
+            }
+        }
+        Text(
+            text = popUpInfo.map { "${it.key}: ${it.value}" }.joinToString("\n"),
+            modifier = Modifier.padding(horizontal = 30.dp, vertical = 16.dp)
+        )
```

Comment:
We could use some layout styling here to display the `popUpInfo` key on the left with it's values on the right similar to the ModalBottomSheet: 


![Screenshot 2024-06-12 at 9 33 28 AM](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/a2160cc2-89b4-4aac-9cc4-c5cf66fcd38e)

Code used: 

```suggestion
        popUpInfo.forEach {
            Row(
                Modifier
                    .fillMaxWidth()
                    .padding(horizontal = 30.dp, vertical = 8.dp), // Could be set on the parent Column as a global padding. 
                horizontalArrangement = Arrangement.SpaceBetween,
                verticalAlignment = Alignment.CenterVertically
            ) {
                Text(text = "${it.key}:", style = MaterialTheme.typography.labelMedium)
                Text(text = "${it.value}")
            }
        }
```

Would like your thoughts on this as well, maybe we could even enhance it with a divider between each entry but I feel it would help clarify that these are the selected GeoElement attributes.

Reply:
Yep I think this looks good!